test/Routing/Dht1Test.cs

```diff
@@ -85,10 +86,10 @@ public async Task ProcessFindNodeMessage_Self()
                 var request = new DhtMessage
                 {
                     Type = MessageType.FindNode,
-                    Key = self.Id.ToArray()
+                    Key = ByteString.CopyFrom(self.Id.ToArray()),
```

Comment:
why is ByteString being used?

Reply:
.proto bytes are represented as ByteString in generated code.  In this version, I just try to convert the c# type to proto type directly.  I think after this PR, we definitely need to define some custom proto for base58 encoded IDs instead of bytes